paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetLoader.kt

```diff
@@ -59,14 +48,26 @@ internal class DefaultCustomerSheetLoader @Inject constructor(
         )
     }
 
-    private suspend fun retrieveElementsSession(): Result<ElementsSession> {
-        return elementsSessionRepository.get(
-            PaymentSheet.InitializationMode.DeferredIntent(
-                PaymentSheet.IntentConfiguration(
-                    mode = PaymentSheet.IntentConfiguration.Mode.Setup(),
-                )
+    private suspend fun retrieveElementsSession(
+        configuration: CustomerSheet.Configuration?,
+    ): Result<ElementsSession> {
+        val initializationMode = PaymentSheet.InitializationMode.DeferredIntent(
+            PaymentSheet.IntentConfiguration(
+                mode = PaymentSheet.IntentConfiguration.Mode.Setup(),
             )
         )
+        return elementsSessionRepository.get(initializationMode).mapCatching { elementsSession ->
+            val billingDetailsCollectionConfig = configuration?.billingDetailsCollectionConfiguration.toInternal()
+
+            lpmRepository.update(
+                stripeIntent = elementsSession.stripeIntent,
+                serverLpmSpecs = elementsSession.paymentMethodSpecs,
+                billingDetailsCollectionConfiguration = billingDetailsCollectionConfig,
+                isDeferred = true,
+            )
+
+            elementsSession.requireValidOrThrow()
```

Comment:
Is this new behavior?

Reply:
This is what we have on master. https://github.com/stripe/stripe-android/blob/master/paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetLoader.kt#L67